Alpaca.Markets/RestClient.Polygon.cs

```diff
@@ -160,6 +160,40 @@ public Task<IEnumerable<IExchange>> ListExchangesAsync()
                 _polygonHttpClient, FakeThrottler.Instance, builder);
         }
 
+        /// <summary>
+        /// Gets list of historical minute bars for single asset from Polygon REST API endpoint.
+        /// </summary>
+        /// <param name="symbol">>Asset name for data retrieval.</param>
+        /// <param name="timespan">>Base length of bars. ("minute" or "day".)</param>
+        /// <param name="multiplier">>Number of bars to combine in each result.</param>
+        /// <param name="dateFromInclusive">Start time for filtering (inclusive).</param>
+        /// <param name="dateToInclusive">End time for filtering (inclusive).</param>
+        /// <param name="unadjusted">Set to true if the results should not be adjusted for splits.</param>
+        /// <returns>Read-only list of minute bars for specified asset.</returns>
+        public Task<IHistoricalItemsV2<IAgg>> ListAggregatesAsyncV2(
+            String symbol,
+            String timespan,
```

Comment:
I dislike this approach with `timespan` parameter passed as string. Add passed value check with exception throwing, use enumeration instead of string,  or create separate methods for days/minutes aggregates (most preferable for me).

Reply:
I'll do the separate methods one for consistency with V1.